index.md

```diff
@@ -63,5 +63,4 @@ Jay A. Spalding Ph.D.
 Licensed psychologist TN and GA
 
 Office: 423-756-2894
-Voicemail: 423-517-7575
```

Comment:
Yes, but he does not want it to be public. Calls will go to the main CBI number and staff will screen his calls for him.